kimchi/src/proof.rs

```diff
@@ -204,6 +215,7 @@ pub mod caml {
         ),
         pub generic_selector: Vec<CamlF>,
         pub poseidon_selector: Vec<CamlF>,
+        pub foreign_mul_selector: Option<(Vec<CamlF>, Vec<CamlF>, Vec<CamlF>)>,
```

Comment:
Because it's a tuple, rather than a vector and it cannot be empty, so needs to be an option.  I think it's expecting a tuple, so didn't want to change this.